### PR TITLE
[LIVE-1317] "something else" choice at repair modal for Nano S

### PR DESCRIPTION
**Pull request by alexandremgo**
## 🦒 Context (issues, jira)

[LIVE-1317](https://ledgerhq.atlassian.net/browse/LIVE-1317?atlOrigin=eyJpIjoiMWJkNTJmYzUyYzc4NGRmZmFlNzEwZDFlMTkwZDUxNWEiLCJwIjoiaiJ9)

The "something else" choice redirects to the end of the usb-troubleshooting flow

Add to adapt a bit compared to the given design (from figma): the Nano S repair modal is old and has not been updated to the new troubleshooting screens.

## 💻  Description / Demo (image or video)

<details>
  <summary>Video: new feature</summary>

https://user-images.githubusercontent.com/15096913/163337621-789599a3-3066-4cf5-ab43-f8fe0d16d1ec.mp4

</details>

<details>
  <summary>Video: the other repair modal screens are not broken</summary>

https://user-images.githubusercontent.com/15096913/163339871-774ec649-5b2c-4929-b752-bc429613c3e8.mp4

</details>

## 🖤  Expectations to reach

- **on QA**: at least one of these two checkboxes must be checked:
  - [x] a specific test planned is defined on Jira 
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

### CODE REVIEW

**Review by thomasrogerlux — APPROVED** [review]
I don't understand why this is specific to Nano S but code looks good

File: src/renderer/modals/RepairModal/index.js [hunk 1]
```diff
@@ -133,13 +133,32 @@ type Props = {
 class RepairModal extends PureComponent<Props, *> {
   state = {
     selectedOption: null,
+    availableRepairChoices: [
+      ...repairChoices,
+      { forceMCU: "", label: "Something else", id: "somethingElse" },
```
Comment (juan-cortes) [inline src/renderer/modals/RepairModal/index.js:138]: Wouldn't this make it not translatable?

**Review by juan-cortes — CHANGES_REQUESTED** [review]

File: src/renderer/modals/RepairModal/index.js (hunk 1 above)
Comment (alexandremgo) [reply src/renderer/modals/RepairModal/index.js:138]: Yes - but I thought that all the labels from `repairChoices`, coming from LLC, were not translated. But actually, i just saw that there is a translation with key `somethingElse` so I'm going to change for this :clock1:

Comment (alexandremgo) [reply src/renderer/modals/RepairModal/index.js]: Updated using a translation key:

<details>
  <summary>New translated label</summary>

Screenshot from 2022-04-21 09-40-21

</details>

Comment (juan-cortes) [reply src/renderer/modals/RepairModal/index.js]: So nothing else is translated :wtf:

Comment (alexandremgo) [reply src/renderer/modals/RepairModal/index.js]: Yes, for this old modal for Nano S, the labels text are coming from [deviceWordings.ts on LLC](https://github.com/LedgerHQ/ledger-live-common/blob/develop/src/deviceWordings.ts) - and used [here in firmwareUpdate-repair.ts](https://github.com/LedgerHQ/ledger-live-common/blob/d63df6a2fadeb0e2c1d49823ec5ed91062d58f11/src/hw/firmwareUpdate-repair.ts#L22)

**Review by juan-cortes — APPROVED** [review]
lgtm

File: src/renderer/components/RepairDeviceButton.js [hunk 2]
```diff
@@ -123,6 +130,7 @@ const RepairDeviceButton: React$ComponentType<Props> = React.forwardRef(function
         desc={t("settings.repairDevice.desc")}
         progress={progress}
         error={error}
+        enableSomethingElseChoice={true}
```
Comment (juan-cortes) [inline src/renderer/components/RepairDeviceButton.js:133]:
```suggestion
        enableSomethingElseChoice
```
https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true

**Review by juan-cortes — APPROVED** [review]
Just nitpicking

**Pull request merged**